torch/_inductor/graph.py

```diff
@@ -854,10 +854,13 @@ def constant_name(self, name: str, device_override: Optional[torch.device]):
         """
         if self.constants[name].device == device_override or device_override is None:
             return name
-        return self.allocate_non_dup_const_name(
-            f"{name}_{device_override.type}{device_override.index or 0}",
-            self.constants[name].to(device_override),
-        )
+        with torch.utils._python_dispatch._disable_current_modes():
```

Comment:
Just want to double-check - would this be too aggressive? Or, we should always have a real tensor at this point.

Reply:
Good question. Constant buffers are specialized, so they really shouldn't be faked.